### PR TITLE
implemented add customer

### PR DESCRIPTION
**Pull request by YongchangLiu98**
Implemented create customer
Modified Files: service/models.py, service/routes.py, tests/test_models.py

### CODE REVIEW

**Comment by ryanjcohen** [comment]
I just connected the create issue to this pull request (the Zenhub board created a separate issue for the pull request review otherwise) and moved the issue over to review

**Comment by Bobby-Zien** [comment]
Hi Yongchang, could you rebase your commits into 1 commit? If you need helps with that, let me know and I'm happy to give you a hand

**Comment by YongchangLiu98** [comment]
> Hi Yongchang, could you rebase your commits into 1 commit? If you need helps with that, let me know and I'm happy to give you a hand

Actually I did and it wo

File: service/models.py [hunk 1]
```diff
@@ -70,15 +70,20 @@ def deserialize(self, data):
             data (dict): A dictionary containing the resource data
         """
         try:
-            self.name = data["name"]
+            self.id=data["id"]
+            self.username = data["username"]
+            self.password=data["password"]
+            self.first_name=data["first_name"]
+            self.last_name=data["last_name"]
+            self.addresses=data["addresses"]
```
[the remark this answers is not in the document]
Comment (YongchangLiu98) [reply service/models.py:84]: I implemented add customer but I didn't handle the cases when the input string is an empty string.
Maybe we will fix it later.

**Pull request closed without merging**